transit_odp/browse/common.py

```diff
@@ -1,18 +1,22 @@
+from typing import Dict
+
+import pandas as pd
+
 from transit_odp.naptan.models import AdminArea
-from transit_odp.otc.models import Service
 from transit_odp.organisation.constants import TravelineRegions
-from typing import Dict
+from transit_odp.otc.models import Service
 
 
 def get_all_naptan_atco_map() -> Dict[str, str]:
```

Comment:
change the return type for this function 
and we can rename the function as it not returning the map or dict

Reply:
Updated